.rhcicd/clowdapp-recipients-resolver.yaml

```
@@ -14,6 +14,7 @@ objects:
     envName: ${ENV_NAME}
     dependencies:
     - rbac
+    - kessel-relations
```

Let's do this until Kessel is stable in prod:
```suggestion
    optionalDependencies:
    - kessel-relations
```